net-irc/inspircd/inspircd-2.0.25.ebuild

```diff
@@ -20,7 +20,7 @@ RDEPEND="
 	geoip? ( dev-libs/geoip )
 	gnutls? ( net-libs/gnutls:= dev-libs/libgcrypt:0 )
 	ldap? ( net-nds/openldap )
-	mysql? ( virtual/mysql )
+	mysql? ( dev-db/mysql-connector-c )
```

Comment:
`:=`?

Reply:
Not sure what I'd be `:=`-ing, the instructions read: "If you need libmysqlclient.so, please depend on dev-db/mysql-connector-c. If you need or can use libmariadb.so, please depend on
dev-db/mariadb-connector-c."